lib/puppet/application/agent.rb

```diff
@@ -124,18 +124,26 @@ def help
 USAGE NOTES
 -----------
 'puppet agent' does its best to find a compromise between interactive
-use and daemon use. Run with no arguments and no configuration, it will
-go into the background, attempt to get a signed certificate, and retrieve
-and apply its configuration every 30 minutes.
+use and daemon use. If you run it with no arguments and no configuration, it
+goes into the background, attempts to get a signed certificate, and retrieves
+and applies its configuration every 30 minutes.
 
-Some flags are meant specifically for interactive use -- in particular,
+Some flags are meant specifically for interactive use --- in particular,
 'test', 'tags' and 'fingerprint' are useful.
 
 '--test' does a single run in the foreground with verbose logging, then exits.
-It will also exit if it can't get a valid catalog. The exit code after running
-with '--test' is 0 if the catalog was successfully applied, and 1 if the run
+It alsos exit if it can't get a valid catalog. The exit code after running
+with '--test' is 0 if the catalog was successfully applied, or 1 if the run
```

Comment:
The `--test` option includes the `--detailed-exitcodes` option. Maybe reword this sentence to reflect that?

The exit code when using `--test` is not a binary option of either `0` or `1`. It could be any of the exit codes specified by the `--detailed-exitcodes` option.

Reply:
Hi @natemccurdy ; I can reword! So you're saying you don't have to pass --detailed-exitcodes at all with --test? It's just inherently included, yeah?

Oof, nice typo I added while I was at it, too. 🥇

Reply:
Yes, that is correct `--test` bundles together multiple other flags. One of which is `--detailed-exitcodes`.